src/main/kotlin/com/sourcegraph/cody/commands/ui/CommandsTabPanel.kt

```diff
@@ -44,48 +50,75 @@ class CommandsTabPanel(
+    fun addSeparator(name: String) {
+      val separator = TitledSeparator(name)
+      separator.maximumSize = Dimension(Int.MAX_VALUE, separator.preferredSize.height)
+      add(separator)
+    }
 
+    addSeparator("Edit Commands")
     if (ConfigUtil.isFeatureFlagEnabled("cody.feature.inline-edits") ||
         CodyApplicationSettings.instance.isInlineEditionEnabled) {
       addInlineEditActionButton(EditCodeAction.ID)
       addInlineEditActionButton(DocumentCodeAction.ID)
       addInlineEditActionButton(TestCodeAction.ID)
     }
 
+    addSeparator("Chat Commands")
+    CommandId.values().forEach { command -> addCommandButton(command) }
+
     IgnoreOracle.getInstance(project).addListener(this)
     FixupService.getInstance(project).addListener(this)
   }
 
   private fun addInlineEditActionButton(actionId: String) {
     val action = ActionManagerEx.getInstanceEx().getAction(actionId)
-    addButton(actionId, action.templatePresentation.text, action.templatePresentation.mnemonic) {
-      val editor = FileEditorManager.getInstance(project).selectedTextEditor ?: return@addButton
-      val dataContext = (editor as? EditorEx)?.dataContext ?: return@addButton
-      val managerEx = ActionManagerEx.getInstanceEx()
-      val event =
-          AnActionEvent(null, dataContext, ActionPlaces.UNKNOWN, Presentation(), managerEx, 0)
-      ActionUtil.performActionDumbAwareWithCallbacks(action, event)
-    }
+    val button =
+        addButton(
+            actionId, action.templatePresentation.text, action.templatePresentation.mnemonic) {
+              val editor =
+                  FileEditorManager.getInstance(project).selectedTextEditor ?: return@addButton
+              val dataContext = (editor as? EditorEx)?.dataContext ?: return@addButton
+              val managerEx = ActionManagerEx.getInstanceEx()
+              val event =
+                  AnActionEvent(
+                      null, dataContext, ActionPlaces.UNKNOWN, Presentation(), managerEx, 0)
+              ActionUtil.performActionDumbAwareWithCallbacks(action, event)
+            }
+
+    button.add(JLabel(Icons.Edit.Beta), BorderLayout.LINE_START)
   }
 
   private fun addCommandButton(commandId: CommandId) {
     addButton(commandId.id, commandId.displayName, commandId.mnemonic) { executeCommand(commandId) }
   }
 
-  private fun addButton(actionId: String, displayName: String, mnemonic: Int, action: () -> Unit) {
+  private fun addButton(
+      actionId: String,
+      displayName: String,
+      mnemonic: Int,
+      action: () -> Unit
+  ): JButton {
     val button = JButton(displayName)
+    buttons[actionId] = button
+
+    button.layout = BorderLayout()
+    button.border = BorderFactory.createEmptyBorder(0, 5, 0, 10)
+    ActionManagerEx.getInstanceEx().getAction(actionId).shortcutSet.shortcuts.firstOrNull()?.let {
+      button.add(JLabel(KeymapUtil.getShortcutText(it)), BorderLayout.LINE_END)
```

Comment:
this looks great (not fully intellij-like but anyway) 🚀

Reply:
Yea, it does not look native but I think discoverability might be worth it, as we have plenty of unused real estate on that buttons :)